ontology/yaml/resources/METERS/entity_types/GM.yaml

```diff
@@ -22,3 +22,13 @@ GM_STANDARD:
   - gas_flowrate_sensor
   - gas_volume_accumulator
   - gas_temperature_sensor
+
+GM_PULSE:
+  id: "01ff9dc3-07ba-4808-86fc-beac0234e97f"
```

Comment:
Remove ID. This is set automatically when the branch is committed.

Reply:
Removed.